### DISCUSSION TITLE
Datarefs/events for Baro Inc / Dec

### DISCUSSION THREAD

**Discussion by studio051** [flybywiresim/aircraft#4836] [Help] [answered]
Any dataref / events like Baro Inc / dec

**Comment by studio051** [comment, marked as answer]
anyone ?

**Reply by tracernz** [reply]
The ones that deviate from the standard MSFS events should be documented in https://github.com/flybywiresim/a32nx/blob/master/docs/a320-events.md

**Reply by tracernz** [reply]
Try `H:A320_Neo_SAI_KNOB_BARO_C` and `H:A320_Neo_SAI_KNOB_BARO_AC`, from:
https://github.com/flybywiresim/a32nx/blob/master/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml#L800

**Reply by studio051** [reply]
Thank you very much , will give it a try in the eve

**Reply by tracernz** [reply]
> Thank you very much , will give it a try in the eve

Derp, that is for the standby instrument. Need to look for the main EFIS panel one..

**Reply by tracernz** [reply]
`K:KOHLSMAN_DEC` and `K:KOHLSMAN_INC`
